src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import todos from '../../api/todos';
+
+type Props = {
+  users: User[],
+  onSubmit: (todo: Todo) => void;
+};
+
+export const NewTodo: React.FC<Props> = ({ users, onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isUserChosen, setIsUserChosen] = useState(true);
```

Comment:
In your case, you may use a single state like `isError` and  calculate the values from other states

Reply:
The problem with calculation is the fact that in the beginning of rendering I need `isTitleEmpty` and `isUserChosen` to be the opposite of their true meaning, so that errors wouldn't be shown at first render

Reply:
> The problem with calculation is the fact that in the beginning of rendering I need `isTitleEmpty` and `isUserChosen` to be the opposite of their true meaning, so that errors wouldn't be shown at first render

It doesn't make sense when we have an empty string as `title` but at the same time `isTitleEmpty` is false 🤷‍♂️ 
The same as I see `selectedUserId` is 0 which means that no user was selected but at the same time `isUserChosen` is true.

If you want to show errors in different places at least you need provide not confusing names for your flags, maybe something like `isTitleValid`, because the title cannot be empty and not empty at the same time.